uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -478,7 +486,7 @@ Arguments:
 sub runCommand {
     my $this = shift;
     my ($command) = @_;
-    print "\n\n $command \n\n " if $this->{'verbose'};
+    print "\n\n $command \n\n " if $this->{verbose};
```

Comment:
You are accessing the verbose attribute of object $this so the quotes must not be there. This is typical Perl OO field access syntax.